fe1-web/webpack.config.js

```diff
@@ -7,7 +7,16 @@ const aliases = {
 };
 
 async function configAsync(env, argv) {
-  const config = await createExpoWebpackConfigAsync(env, argv);
+  const config = await createExpoWebpackConfigAsync({
+      ...env,
+      babel: {
+          // This is a workaround for https://github.com/react-native-elements/react-native-elements/issues/3607
+          // to get the new version of react-native-elements working
+          // Revert this commit as soon as the new version is properly released and this
+          // issue is resolved
```

Comment:
You might also want to add a GitHub issue in our own repository to track this, documenting the rc6 + hack😄